Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+## Coaching Guide
+
+_Original author: Diana Vashti_.
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments or most of an afternoon doing code reviews off “Prrr”. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
+
+## How do I know if I am coaching?
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
```

Comment:
you know you're coaching if you are on goal 135 and jrob, the coaching coordinator, tells you you're coaching.